packages/frontend/src/components/SchedulersView/index.tsx

```diff
@@ -0,0 +1,121 @@
+import { ActionIcon, Group, Tabs, Title, Tooltip } from '@mantine/core';
+import { IconClock, IconRefresh, IconSend } from '@tabler/icons-react';
+import React, { FC } from 'react';
+import { useQueryClient } from 'react-query';
+import { useSchedulerLogs } from '../../hooks/scheduler/useScheduler';
+import { useTableTabStyles } from '../../hooks/styles/useTableTabStyles';
+import useToaster from '../../hooks/toaster/useToaster';
+import MantineIcon from '../common/MantineIcon';
+import ResourceEmptyState from '../common/ResourceView/ResourceEmptyState';
+import { SettingsCard } from '../common/Settings/SettingsCard';
+import Logs from './LogsView';
+import Schedulers from './SchedulersView';
+
+const SchedulersView: FC<{ projectUuid: string }> = ({ projectUuid }) => {
+    const { data } = useSchedulerLogs(projectUuid);
+    const queryClient = useQueryClient();
+    const tableTabStyles = useTableTabStyles();
+    const { showToastSuccess } = useToaster();
+    const emptyState = (
+        <ResourceEmptyState
+            title="No scheduled deliveries on this project"
+            description="Go to a chart or dashboard to set up your first scheduled delivery"
+        />
+    );
+    const logsEmptyState = (
+        <ResourceEmptyState
+            title="Scheduled deliveries have not run any jobs as of now"
+            description="Check in later of hit the refresh button to see if any jobs have run"
+        />
+    );
```

Comment:
you can extract these out of this function to avoid calling it each time component rerenders